e2e-tests/version-service/conf/crd.yaml

```diff
@@ -31504,18 +31516,12 @@ spec:
       storage: true
       subresources:
         status: {}
-status:
```

Comment:
Sorry, but I can't see how we are losing this field with these changes. I'm probably missing something.

Reply:
The `crd.yaml` file is generated using the `e2e-tests/manifest` script. In this script we use the [`controller-gen`](https://book.kubebuilder.io/reference/controller-gen.html) tool for generating it. Old crd files were generated with v.0.8.0 version of controller-gen, but in this PR I used v0.9.0 version. The newer version doesn't generate this field

Reply:
Aha, ok. I thought it was because of the version bump but just wanted to confirm. Thanks!